docs/meta.md

```diff
@@ -77,9 +77,9 @@ The following keys are optional:
 
 * `uses`: a map of names and skills
```

Comment:
* `slots`: a map of interfaces

We need more details here at some point, that'll do for now.